corehq/apps/export/models/new.py

```diff
@@ -105,32 +105,44 @@ def create_default_from_export_item(group_schema_path, item, build_ids_and_versi
 
         :param group_schema_path: The path of the group_schema that the item belongs to
         :param item: An ExportItem instance
-        :param app_version: The app version that the column export is for
+        :param build_ids_and_versions: A dictionary of build ids that map to versions of that represents the
         :returns: An ExportColumn instance
         """
 
+        is_main_table = group_schema_path == MAIN_TABLE
+
+        column = ExportColumn(
+            item=item,
+            label=item.label,
+        )
+        column.update_properties_from_build_ids(build_ids_and_versions)
+        column.selected = not column._is_deleted(build_ids_and_versions) and is_main_table
+        return column
+
+    def _is_deleted(self, build_ids_and_versions):
         is_deleted = True
         for app_id, version in build_ids_and_versions.iteritems():
-            if item.last_occurrences.get(app_id) == version:
+            if self.item.last_occurrences.get(app_id) == version:
                 is_deleted = False
                 break
+        return is_deleted
 
-        is_main_table = group_schema_path == MAIN_TABLE
+    def update_properties_from_build_ids(self, build_ids_and_versions):
```

Comment:
this basically ensures that the column is "up-to-date" with any changes that might've happened